ceph.spec.in

```diff
@@ -1035,11 +1035,22 @@ rm -rf %{buildroot}
 %endif
 %{_unitdir}/ceph-disk@.service
 %{_unitdir}/ceph.target
+%if 0%{with python2}
 %{python_sitelib}/ceph_detect_init*
 %{python_sitelib}/ceph_disk*
+%else
+%{python3_sitelib}/ceph_detect_init*
```

Comment:
is it possible to use %{python%{_python_buildid}_sitelib} ? i.e macro in macro name?

Reply:
yes, I believe it is possible, and I see where you're going with that question - it would allow us to eliminate the conditional in this case, but if you look at how `%{python_sitelib}` and `%{python3_sitelib}` are conditionalized elsewhere in the spec file, it wouldn't be consistent.

In other words, we could do this near the top of the spec file: `%global _python_sitelib %{python%{_python_buildid}_sitelib}` and then replace `%python_sitelib` and `%python3_sitelib` with `%_python_sitelib`. And that would allow us to get rid of the ceph_disk and ceph_volume conditionals, but none of the others - e.g. https://github.com/ceph/ceph/blob/master/ceph.spec.in#L1160-L1168

I'd rather keep the usage consistent throughout the spec file.